src/Browser/Chromium/VersionResolver.php

```diff
@@ -37,9 +46,15 @@ public function from(OperatingSystem $operatingSystem, string $path): Version
         }
 
         if ($operatingSystem->equals(OperatingSystem::WINDOWS())) {
-            return $this->getVersionFromCommandLine(
-                sprintf('wmic datafile where name="%s" get Version /value', $path)
-            );
+            foreach (self::getWindowsCommandsForVersion() as $possibleCommand) {
+                try {
+                    return $this->getVersionFromCommandLine($possibleCommand);
+                } catch (InvalidArgumentException $exception) {
+                    // @ignoreException
```

Comment:
Can you throw a new exception here with `$exception` as the `$previous` parameter?